src/Modules/Visualization/ShowField.cc

```diff
@@ -240,81 +241,45 @@ GeometryHandle ShowFieldModule::buildGeometryObject(
   const int dim = field->vmesh()->dimensionality();
   if (showEdges && dim < 1) { showEdges = false; }
   if (showFaces && dim < 2) { showFaces = false; }
+  //adjust color map to scalar data if there is one.
```

Comment:
@brigb123 Finally, is this PR supposed to completely implement the RescaleColorMap module? If so, this block should just be removed--ShowField does not do any min/max scaling. If you intend to remove this section separately, let me know.